q2_demux/tests/test_demux.py

```diff
@@ -809,12 +820,22 @@ def test_inconsistent_sequence_length_paired(self):
         barcode_map = qiime2.MetadataCategory(barcode_map)
 
         demux_data = emp_paired(bpsi, barcode_map)
-        with tempfile.TemporaryDirectory() as output_dir:
-            with self.assertRaisesRegex(ValueError,
-                                        'Observed sequences of length'):
+        lengths = [1, 3, 5, 7]
+        for n in range(1, 5):
+            with tempfile.TemporaryDirectory() as output_dir:
+                _lengths = lengths[0:5-n] if n < 4 else [1]
                 # TODO: Remove _PlotQualView wrapper
                 summarize(output_dir, _PlotQualView(demux_data,
-                                                    paired=True), n=2)
+                                                    paired=True), n=n)
+                plot_fp = os.path.join(output_dir, 'data.jsonp')
+                with open(plot_fp, 'r') as fh:
+                    text = fh.read()
+                    text = text.replace('app.init(', '[').replace(');', ']')
+                    jsonp = json.loads(text)[0]
```

Comment:
If you wind up changing `jsonp` above, don't forget to do the same here. You're call, whatever you think is more clear to someone reading this test.